QuickOSM/core/process.py

```diff
@@ -66,8 +68,12 @@ def open_file(
             if not prefix_file:
                 prefix_file = layer_name
 
-            outputs[layer] = join(
-                output_dir, prefix_file + "_" + layer + ".geojson")
+            if output_format == Format.GeoPackage or output_format == Format.Kml:
+                outputs[layer] = join(
+                    output_dir, prefix_file + "." + output_format.value.extension)
+            else:
```

Comment:
Can you make the same here with `elif`
````
else:
    Raise NotImplementedError
```

It's high probability that we will have more formats. We could even make it dynamically generated this list but let's keep that for later.
So by having `NotImplemented` we will know next time where the code has not been updated.